ui/src/app/pages/workspace/workspace-research-summary.tsx

```diff
@@ -1,111 +1,40 @@
-import {FlexRow} from 'app/components/flex';
-import {TextArea} from 'app/components/inputs';
-import colors from 'app/styles/colors';
-import {colorWithWhiteness} from 'app/styles/colors';
-import {reactStyles} from 'app/utils';
+import {TextAreaWithLengthValidationMessage} from 'app/components/inputs';
 import * as React from 'react';
 import {WorkspaceEditSection} from './workspace-edit-section';
 import {ResearchPurposeQuestion} from './workspace-edit-text';
 
-const styles = reactStyles({
-  textArea: {
-    height: '15rem',
-    resize: 'none',
-    width: '48rem',
-    borderRadius: '3px 3px 0 0',
-    boderColor: colorWithWhiteness(colors.dark, 0.5)
-  },
-  textBoxCharRemaining: {
-    justifyContent: 'space-between',
-    width: '48rem',
-    backgroundColor: colorWithWhiteness(colors.primary, 0.95),
-    fontSize: 12,
-    colors: colors.primary,
-    padding: '0.25rem',
-    borderRadius: '0 0 3px 3px', marginTop: '-0.5rem',
-    border: `1px solid ${colorWithWhiteness(colors.dark, 0.5)}`
-  },
-});
 
 interface Props {
+  id: string;
   index: string;
   onChange: Function;
   researchPurpose: ResearchPurposeQuestion;
   researchValue: string;
-  rowId: string;
 }
 
-interface State {
-  showWarningMessage: boolean;
-  researchValue: string;
-  textColor: string;
-}
-
-export class WorkspaceResearchSummary extends React.Component<Props, State > {
+export class WorkspaceResearchSummary extends React.Component<Props> {
```

Comment:
Curious should this be a const rather than class?

Reply:
Ooh, yeah, now that it has no more state it can be a const!